docs/en/about-appium/intro.md

```diff
@@ -73,18 +93,22 @@ the server will start up the automation session and respond with a session ID
 which is used for sending further commands.
 
 **Desired Capabilities**<br/>
-Desired capabilities are a set of keys and values (i.e.,
-a map or hash) sent to the Appium server to tell the server what kind of
-automation session we're interested in starting up. There are also various
-capabilities which can modify the behavior of the server during automation.
-For example, we might set the `platformName` capability to `iOS` to tell
-Appium that we want an iOS session, rather than an Android or Windows one. Or we might
-set the `safariAllowPopups` capability to `true` in order to ensure that,
-during a Safari automation session, we're allowed to use JavaScript to open
-up new windows. See the [capabilities doc](/docs/en/writing-running-appium/caps.md) for the complete list of capabilities available for Appium.
+Desired capabilities are a set of keys and values (i.e., a map or hash) sent to
+the Appium server to tell the server what kind of automation session we're
+interested in starting up. There are also various capabilities which can modify
+the behavior of the server during automation.  For example, we might set the
+`platformName` capability to `iOS` to tell Appium that we want an iOS session,
+rather than an Android or Windows one. Or we might set the `safariAllowPopups`
+capability to `true` in order to ensure that, during a Safari automation
+session, we're allowed to use JavaScript to open up new windows. See the
+[capabilities doc](/docs/en/writing-running-appium/caps.md) for the complete
+list of capabilities available for Appium.
 
 **Appium Server**<br/>
-Appium is a server written in Node.js. It can be built and installed [from source](https://github.com/appium/appium/blob/master/docs/en/contributing-to-appium/appium-from-source.md) or installed directly from NPM:
+Appium is a server written in Node.js. It can be built and installed [from
+source](https://github.com/appium/appium/blob/master/docs/en/contributing-to-appium/appium-from-source.md)
```

Comment:
actually i think they used to need to be, because we had this doc pulled out on our website as the "Intro" doc. but I think we should do away with that and just point to the version in our docs instead.